azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/version.py

```diff
@@ -1,7 +0,0 @@
-# -------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License in the project root for
-# license information.
-# --------------------------------------------------------------------------
-
-VERSION = "1.0.0b8"
```

Comment:
Where are we setting the version instead?

Reply:
Was this supposed to be a _version rename?

Reply:
https://github.com/microsoft/ApplicationInsights-Python/blob/main/azure-monitor-opentelemetry-distro/setup.py#L40

Reply:
Got it. We never deleted the old one.